config/201-clusterrole.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cloud-run-events-controller
+  labels:
+    events.cloud.run/release: devel
+rules:
+
+- apiGroups:
+  - events.cloud.run
+  resources:
+  - gcppubsubsources
+  verbs: &everything
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+
+- apiGroups:
+  - events.cloud.run
+  resources:
+  - gcppubsubsources/status
+  verbs:
+  - get
+  - update
+  - patch
+
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs: *everything
+
+- apiGroups:
+  - ""
+  resources:
```

Comment:
Does it really need global secrets access?

Reply:
good question